src/test/java/Hw16.java

```diff
@@ -0,0 +1,58 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Hw16 {
+    
+    @Test
+    public static void creatingNewPlaylist() throws InterruptedException {
+
+
+        String url = "https://bbb.testpro.io/";
+        String email = "holostenco.yuliya@gmail.com";
+        String password = "te$t$tudent";
+        String myPlaylistName = "Iulia's Playlist";
+
+        WebDriver driver = new ChromeDriver();
+
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.get(url);
+        driver.manage().window().maximize();
+
+        WebElement emailField = driver.findElement(By.xpath("//input[@type='email']"));
+        emailField.click();
+        emailField.sendKeys(email);
+
+        WebElement passwordField = driver.findElement(By.xpath("//input[@type='password']"));
+        passwordField.click();
+        passwordField.sendKeys(password);
+
+        WebElement submitButton = driver.findElement(By.xpath("//button[@type='submit']"));
+        submitButton.click();
+
+        WebElement createANewPlaylistElement = driver.findElement(By.xpath("//*[@title='Create a new playlist']"));
+        createANewPlaylistElement.click();
```

Comment:
I recommend add the "Thread.sleep(2000);" after the web element has loaded, and then you can add a click method

For example
        ```

>         WebElement createANewPlaylistElement = driver.findElement(By.xpath("//*[@title='Create a new playlist']"));
>         Thread.sleep(2000);
>         createANewPlaylistElement.click();

         ```

Error when we don't use wait

![image](https://user-images.githubusercontent.com/104761482/197384595-894d6405-b534-4534-ba07-d8d9b012d4a0.png)